cmd/k8s-operator/proxygroup.go

```diff
@@ -813,6 +813,7 @@ func (r *ProxyGroupReconciler) ensureConfigSecretsCreated(ctx context.Context, p
 					KubeAPIServer: &conf.KubeAPIServer{
 						AuthMode: opt.NewBool(isAuthAPIServerProxy(pg)),
 					},
+					ServerURL: &r.loginServer,
```

Comment:
nit: `r.loginServer` could be `""` - I think we should prefer to only set this if it's non-empty.